src/views/FacilityDetails.vue

```diff
@@ -161,27 +161,31 @@
           </ion-card>
 
           <ion-card>
-            <ion-card-header>
```

Comment:
Please revert all the changes those are not need for this issue to be fixed, like the indentations